packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -238,7 +238,7 @@ class SwitchListTile extends StatelessWidget {
 
   /// The color to use on the track when this switch is on.
   ///
-  /// Defaults to [ThemeData.toggleableActiveColor] with the opacity set at 50%.
+  /// Defaults to [ThemeData.colorScheme.secondary] with the opacity set at 50%.
```

Comment:
I'm not sure that these doc references work. Can simplify to `[ColorScheme.secondary]`